src/sdk/account_sdk.js

```diff
@@ -73,11 +74,15 @@ class AccountSDK {
         }
     }
 
-    // copied from function in sts_sdk
     authorize_request_account(req) {
         const token = this.get_auth_token();
         // If the request is signed (authenticated)
         if (token) {
+            try {
+                signature_utils.check_access_key_status(token.access_key, this.requesting_account);
+            } catch (err) {
+                throw new IamError(IamError.InvalidClientTokenIdInactiveAccessKey);
```

Comment:
why is this here and not in RPC_ERRORS_TO_IAM in iam_rest? I don't think we need to catch it here at all.

Reply:
It is there, but I mapped the `INVALID_ACCESS_KEY_ID` to `IamError.InvalidClientTokenId` and not to `IamError.InvalidClientTokenIdInactiveAccessKey`, and then the message will look different.

Reply:
So just throw another RPC error code for it.

Reply:
I will, thanks.